src/ocean/text/convert/Memory.d

```diff
@@ -87,7 +87,7 @@ public char[] memoryToHexAscii ( void[] mem, char[] output = null )
                 c = '.';
             ascii[idx] = c;
         }
-        Format.format(output, "{}\n", ascii[0 .. idx]);
```

Comment:
You can revert this whole file, it's deprecated (https://github.com/sociomantic-tsunami/ocean/pull/174)